File: Solutions/JavaScript/pattern004.js
// Pattern Programs - 004 [JavaScript]
// Github Repository: https://github.com/mxsena/pattern-programs
let pattern = ""

for (let i = 5; i >= 1; i--) {
    for (let j = i; j <= 5; j++) {
        pattern += i + " "
    }
    pattern += "\n"
}
console.log(pattern)

File: Solutions/C/pattern004.c
// Pattern Programs n.004 [C]
// Github Repository: https://github.com/mxsena/pattern-programs
#include <stdio.h>

int main(void)
{
    int i, j;

    for (i = 5; i >= 1; i--)
    {
        for (j = 1; j <=5; j++) 
        {
            printf("%d ", i);
        }
        printf("\n");
    }
    return 0;
}